src/test/java/bio/terra/janitor/app/controller/JanitorApiControllerTest.java

```diff
@@ -220,7 +220,7 @@ public void getResources() throws Exception {
   }
 
   @Test
-  public void abandonThenBumpResources() throws Exception {
+  public void abandoResources() throws Exception {
```

Comment:
What's the difference between this test and `bumpResources` test? Also, see other comment.

Reply:
Done